docs/examples/scenes-bot.ts

```diff
@@ -19,7 +25,7 @@ echoScene.command('back', leave())
 echoScene.on('text', (ctx) => ctx.reply(ctx.message.text))
 echoScene.on('message', (ctx) => ctx.reply('Only text messages please'))
 
-const bot = new Telegraf(process.env.BOT_TOKEN)
+const bot = new Telegraf<SceneContext>(process.env.BOT_TOKEN)
```

Comment:
It is necessary to extend the bot context in one way or another if scenes should be used.

The default `SceneContext` interface can be used (or `extend`ed) to automatically get the correct types for `ctx.scene` on the context object. If it is omitted, only the regular `Context` properties are available, and hence
```ts
bot.command('greeter', (ctx) => ctx.scene.enter('greeter'))
```
would fail.